requirements/requirements.txt

```diff
@@ -1,10 +1,10 @@
 dataclasses
 ovos-lingua-franca>=0.4.4a2
-deepspeech>=0.9.3
 tensorflow>=2.3.0
 ovos-plugin-manager~=0.0.2
 ovos-utils~=0.0.12
 neon_speech
 neon-utils~=0.15,>=0.15.1a18
 wave
 huggingface-hub
+deepspeech
```

Comment:
Can this be replaced with the Coqui STT package? (https://pypi.org/project/stt/)

Reply:
When I try to run it with stt.Model() it gives me an error:

Reply:
Got it. The error looks similar to the DeepSpeech ones when loading an old/unsupported model. We might need to move to the new package for newer models, but that can be a later PR